xsl/pretext-common.xsl

```diff
@@ -10934,6 +10943,34 @@ http://andrewmccarthy.ie/2014/11/06/swung-dash-in-latex/
     <xsl:value-of select="concat($percent,'%')" />
 </xsl:template>
 
+<!-- ################# -->
+<!-- String Utilities -->
+<!-- ################# -->
+
+<!-- Find a delimiter: find a character that can be wrapped around a string -->
+<!-- Take a string as one input and a list of characters as another input   -->
+<!-- Return the first character from the list that is not in the string     -->
+<xsl:template name="find-unused-character">
+    <xsl:param name="string"/>
+    <xsl:param name="characters" select="concat($apos,'&quot;|/!?=~')"/>
+    <xsl:choose>
+        <xsl:when test="string-length($characters) &lt; 1">
+            <xsl:message>PTX WARNING:  unable to find a character that is not used in "<xsl:value-of select="$string" />" from list provided. Proceeding with default '~'</xsl:message>
```

Comment:
You can put a `&#xa;` in here to get it to split lines when output.  Or not.

Do you want to save initial value of `$characters` in a variable and then repeat it in the warning, rather than just "list provided'?

Reply:
Lines selected are teh last ones shown, preceding lines are automatic from GitHub.

Reply:
"list provided" is a remnant from an earlier version, before I gave a default for `$characters`. So yes, I'll make that change too.